docs/how-to/integrate_sdcore_with_observability.md

```diff
@@ -1,104 +1,131 @@
 # Integrate SD-Core with Canonical Observability Stack
 
-[Charmed Aether SD-Core Terraform modules][Charmed Aether SD-Core Terraform modules] come with built-in support for the Canonical Observability Stack (COS).
-By default, COS deployment and integration is disabled.
-This guide covers two ways of integrating SD-Core with COS:
-1. [Integrating SD-Core with COS at the deployment stage](#option-1)
-2. [Integrating COS with an existing SD-Core deployment](#option-2)
+One of the key aspects considered while developing Charmed Aether SD-Core was making it easily observable.
+To achieve this, each [Charmed Aether SD-Core Terraform module][Charmed Aether SD-Core Terraform modules] includes Grafana Agent application, which allows for integration with the Canonical Observability Stack (COS).
 
-(option-1)=
-## Integrating SD-Core with COS at the deployment stage
+This how-to guide outlines the process of integrating Charmed Aether SD-Core with COS.
 
-This option allows deploying COS and integrating it with SD-Core as a Day 1 operation. 
-
-### Pre-requisites
-
-- A Kubernetes cluster capable of handling the load from both SD-Core and COS
-- [Charmed Aether SD-Core Terraform modules][Charmed Aether SD-Core Terraform modules] Git repository cloned onto the Juju host machine
-
-### Including COS integration in the SD-Core deployment
-
-Inside the directory of a desired SD-Core Terraform module, create `variables.tfvars` file and add following line(s) to it:
-
-```console
-deploy_cos = true
-cos_model_name = "YOUR_CUSTOM_COS_MODEL_NAME" (Optional. Defaults to `cos-lite`.)
-cos_configuration_config = {} (Optional. Allows customization of the `COS Configuration` application.)
-```
+Steps described in this guide can be performed as both Day 1 and Day 2 operations.
 
 ```{note}
-If you have already created the `.tfvars` file, to customize the deployment of SD-Core, you should edit the existing file rather than create a new one.
+Deploying Canonical Observability Stack will increase the resources consumption on the K8s cluster. 
+Make sure your Kubernetes cluster is capable of handling the load from both Charmed Aether SD-Core and COS before proceeding.  
 ```
 
-Proceed with the deployment.
+## 1. Add COS to the solution Terraform module
 
-(option-2)=
-## Integrating COS with an existing SD-Core deployment
-
-This option allows deploying COS and integrating it with SD-Core as a Day 2 operation.
-
-### Pre-requisites
-
-- A Kubernetes cluster capable of handling the load from both SD-Core and COS
-- Any [Charmed Aether SD-Core Terraform module][Charmed Aether SD-Core Terraform modules] deployed
-
-### Adding COS to an existing SD-Core deployment
-
-Go to a directory from which SD-Core was deployed (the one containing Terraform's `.tfstate` file).
-Edit the `.tfvars` and add following line(s) to it:
+Update your solution Terraform module (here it's named `main.tf`):
```

Comment:
Instead of `solution Terraform module`, can we say `target Terraform module` ?  WDYT ?

Reply:
"Solution module" is the name defined by the [CC006 - Terraform modules organization in the Juju ecosystem](https://docs.google.com/document/d/1EG71A2pJ244PQRaGVzGj7Mx2B_bzE4U_OSqx4eeVI1E/edit).